src/pmi/pmi.cs

```diff
@@ -174,7 +174,7 @@ public CustomLoadContext(string assemblyPath)
         // Add in current assembly path and framework path
         pmiPath += Path.GetDirectoryName(assemblyPath);
         pmiPath += ";";
-        pmiPath += Path.GetDirectoryName(typeof(object).Assembly.Location);
+        pmiPath += Path.GetDirectoryName(System.AppContext.BaseDirectory);
```

Comment:
I think this change should be reverted, I think it is going to give the directory of PMI but it should be the directory of the libraries for the runtime under test.

Reply:
```suggestion
        pmiPath += Path.GetDirectoryName(typeof(object).Assembly.Location);
```